modules/task/task.go

```diff
@@ -98,10 +87,3 @@ type Config struct {
 	broker  string `yaml:"broker"`
 	timeout string `yaml:"timeout"`
 }
-
-type backendRegisterFn func(service.Host, Config) (Backend, error)
-
-// RegisterAsyncBackend registers the backend for the async task module
-func RegisterAsyncBackend(backendRegisterFn backendRegisterFn) {
```

Comment:
Discussed this with you yesterday. Decided to make it a more explicit dependency injection to avoid traps when the function has not been set.